pennylane/ops/qubit/parametric_ops.py

```diff
@@ -1269,6 +1269,182 @@ def control_wires(self):
         return self.wires[0:1]
 
 
+class PCPhase(Operation):
+    r"""
+    A Projector Controlled Phase gate.
+
+    .. math:: \Pi_{\phi}(\phi, dim=2, wires=[0,1]) = \begin{bmatrix}
+                e^{i\phi} & 0 & 0 & 0 \\
+                0 & e^{i\phi} & 0 & 0 \\
+                0 & 0 & 1 & 0 \\
+                0 & 0 & 0 & 1
+            \end{bmatrix}.
+
+    Args:
+        phi (float): rotation angle :math:`\phi`
+        dim (int): the qubit subspace on which we apply the phase shift
+        wires (Any, Wires): the wire the operation acts on
```

Comment:
Is the wires type correct? Seems inconsistent with the example given below.

Reply:
Nice catch!